english/dig/explore-outside/explore-outside.md

```diff
@@ -0,0 +1,6 @@
+
+Finally you make it outside of all that ridiculous marshmellow.  So much white.. so much sugar!  Look at the incredible valley that spreads out before you.  The distance holds a large mountain which shines with a silvery pink sheen as the sun rises above the ridgeline.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.